### DISCUSSION TITLE
Customization DB Group

### DISCUSSION THREAD

**Discussion by arashsaffari** [codeigniter4/shield#727] [Ideas]
It would be great if you could select which database group to use in the configuration. In **Config/Auth.php**, the following should be added:

```php
public string $DBGroup = '';
```

Additionally, the following should be added in **Model/BaseModel.php**:

```php
protected function initialize(): void
{
  $authConfig = config('Auth');
  $this->DBGroup = $authConfig->DBGroup;
}
```

**Comment by datamweb** [comment]
@arashsaffari , I edited your sample code. In my opinion, the addition of this feature is good. 
**+1 from me.**

**Comment by datamweb** [comment]
This feature was added in #744.
